README.md

```diff
@@ -48,6 +48,33 @@ $ sudo dnf install mpris-scrobbler
 First, install the correct repository for your operating system from COPR.
 Then, install the `mpris-scrobbler` package with your package manager of choice.
 
+### Ubuntu 18.04
+
+Install the dependencies:
+
+    sudo apt install libevent-2.1-6 libevent-dev libdbus-1-dev dbus dbus-user-session libcurl4 libcurl4-openssl-dev libjson-c-dev libjson-c3 meson
+
+Restart after this to allow `dbus-user-session` to take effect.
+
+Clone the repo, then follow the instructions under **Compile from source**.
+
+For initial sign-up it behaves a little weirdly so **Usage** is slightly different the first time when signing onto a service:
```

Comment:
Could you please remove the section about setting up? It has nothing to do with installing. 

If you are sure that the sign-on requires different steps on Ubuntu, please open an issue about it so we can check it together, altough I'm not sure why it would be so.